### PR TITLE
Adds timer to flip to avoid IOS double click

### PR DESCRIPTION
**Pull request by nihilence**

### CODE REVIEW

File: src/views/Flipper.js [hunk 1]
```diff
@@ -8,11 +8,11 @@
  */
 
 define(function(require, exports, module) {
-    var Transform = require('../core/Transform');
-    var Transitionable = require('../transitions/Transitionable');
-    var RenderNode = require('../core/RenderNode');
-    var OptionsManager = require('../core/OptionsManager');
-
+    var Transform = require('famous/core/Transform');
```
Comment (MylesBorins) [inline src/views/Flipper.js]: please leave relative path's.  This is done due to allow path resolving to be done explicitly.

**Comment by MylesBorins** [comment]
Please sign our CLA so that we can merge this --> https://famo.us/cla

Also you will notice that this did not pass travis, please fix all style errors.

**Comment by MylesBorins** [comment]
As well, I'm not 100% sure that this will land.  It seems odd to modify the api of flipper simply for IOS.

thoughts @michaelobriena

**Comment by nihilence** [comment]
Alright, well... I am sorry that it didn't meet the build. I've never done
this before. That's just what I did to fix it for my personal project. I
can modify it soon when I get some time.

On Tue, Apr 7, 2015 at 11:43 PM, Myles Borins notifications@github.com
wrote:

> As well, I'm not 100% sure that this will land. It seems odd to modify the
> api of flipper simply for IOS.
> 
> thoughts @michaelobriena https://github.com/michaelobriena
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Famous/famous/pull/672#issuecomment-90821512.

## 

~Sabice

**Comment by MylesBorins** [comment]
It is super awesome that you are contributing! The style errors are pretty straight forward to fix 

```
src/views/Flipper.js
  15:49  error  Missing semicolon            semi
  61:1   error  Trailing spaces not allowed  no-trailing-spaces
  64:27  error  Missing semicolon            semi
  65:1   error  Trailing spaces not allowed  no-trailing-spaces
```

Basically add two semi colons and remove some spaces.

What might be harder to merge in is a fix relying on setTimeout that is platform specific.  BUT you have uncovered exactly what the problem is.  I have a feeling like @michaelobriena @marklu or @alexanderGugel would have some thoughts on this.

No matter the outcome please keep contributing, we wouldn't know about these problems otherwise!

**Comment by nihilence** [comment]
Well, the reason I did it with a timer is because regardless of the
platform, we wanted the full transition to be made before a user could flip
again. We have tried it on iOS and Android emulators and real devices and
this works for what we need. Sorry if you can't use it.

On Wed, Apr 8, 2015 at 12:21 AM, Myles Borins <notifications@github.com>
wrote:

> It is super awesome that you are contributing! The style errors are pretty
> straight forward to fix
>
> src/views/Flipper.js
>   15:49  error  Missing semicolon            semi
>   61:1   error  Trailing spaces not allowed  no-trailing-spaces
>   64:27  error  Missing semicolon            semi
>   65:1   error  Trailing spaces not allowed  no-trailing-spaces
>
> Basically add two semi colons and remove some spaces.
>
> What might be harder to merge in is a fix relying on setTimeout that is
> platform specific. BUT you have uncovered exactly what the problem is. I
> have a feeling like @michaelobriena <https://github.com/michaelobriena>
> @marklu <https://github.com/marklu> or @alexanderGugel
> <https://github.com/alexanderGugel> would have some thoughts on this.
>
> No matter the outcome please keep contributing, we wouldn't know about
> these problems otherwise!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Famous/famous/pull/672#issuecomment-90828671>.
>



-- 
~Sabice

**Comment by MylesBorins** [comment]
Thank you for sending this PR. We are genuinely sorry it did not make it into the repo. Getting mixed mode out the door involved moving all of our resources off of this project. As such we will not be merging any sort of enhancement or major bug fix.

We would love to have you join us over at the [famous engine](http://github.com/famous/engine), to test the edges in a environment where we can assure you prompt response to PR's and a constant development cycle

**Pull request closed without merging**